.github/workflows/build.yml

```diff
@@ -0,0 +1,74 @@
+name: Bump, Tag, and Publish
+# The purpose of the workflow is to:
+#  1. Bump the version number and tag the release if not a PR
+#  2. Build docker image and publish to GCR
+#
+# When run on merge to main, it tags and bumps the patch version by default. You can
+# bump other parts of the version by putting #major, #minor, or #patch in your commit
+# message.
+#
+# When run on a PR, it simulates bumping the tag and appends a hash to the pushed image.
+#
+# The workflow relies on github secrets:
+# - GCP_PUBLISH_EMAIL - SA email for publishing to dsp-artifact-registry
+# - GCP_PUBLISH_KEY_B64 - SA key (Base64-encoded JSON string) for publishing to dsp-artifact-registry
+# - BROADBOT_TOKEN - the broadbot token, so we can avoid two reviewer rule on GHA operations
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'README.md'
```

Comment:
Is it the only file to ignore? Is there any generated `*.log` file?

Reply:
Added `**.log` to `paths-ignore` just incase although it's unlikely to have a log file generated for this repo. Unit test done for this workflow. The results can be seen in the Actions log.